### DISCUSSION TITLE
error when compile EzEngine

### DISCUSSION THREAD

**Discussion by bendico1** [ezEngine/ezEngine#1116] [General]
Hi,
I'm a newcomer here. I follwed the instructions to build EzEngine from source, but when I try to compile with vs2022, an error is tossed out: LINK2001 DXGI_DEBUG_ALL unresolved external symbol.
What's wrong? How I can solve the issue?
However, if I use the precompiled version, the editor runs without any problem.
Thanks in advance, kind regards.

                                   Cosmo
![EzEngine not correctly compiled](https://github.com/ezEngine/ezEngine/assets/2538724/1699b6d5-6e2c-430d-9fa5-e690f4fc6e0f)

**Comment by jankrassnigg** [comment]
I would guess that you don't have the necessary workload in visual studio installed. See this page https://ezengine.net/pages/docs/build/build-windows.html#microsoft-visual-studio

**Comment by bendico1** [comment]
Screenshots of  my VS_Installer page: 

![Vs_Installer](https://github.com/ezEngine/ezEngine/assets/2538724/6b979dee-72c3-4ee3-bc2b-f3a6a096f292)
![Vs_Installer2](https://github.com/ezEngine/ezEngine/assets/2538724/bb20a8ae-ba52-4578-861b-7df598026190)

**Comment by jankrassnigg** [comment]
As the document says, you can use that config file to configure the installer.
Also for these kind of errors, Google might be able to help.
